OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -232,7 +236,10 @@ public virtual Barrel CheckFire(Actor self, IFacing facing, Target target)
 
 			ticksSinceLastShot = 0;
 
-			var barrel = Barrels[Burst % Barrels.Length];
+			// If Weapon.Burst == 1, cycle through all LocalOffsets, otherwise use the offset corresponding to current Burst
```

Comment:
Can you please document this in the `LocalOffset` desc too?

Reply:
Done.